_docs_explore/64-request-items.md

```diff
@@ -1,4 +1,365 @@
+| Identity       | Sender          | N         | `N/A`           | Use ShareAttributeRequestItem instead.                                                                                                                                                                                                                                |
+| Identity       | Recipient       | Y         | `USER_DECISION` | University sends student his certificate (Propose would be inappropriate in this case, because the student should not be able to return his own value)                                                                                                                |
+| Identity       | `<empty>`       | Y         | `USER_DECISION` | An empty owner defaults to an Attribute with owner=`<recipient>`. This is needed for Requests inside of Relationship Templates, since you don’t know the Enmeshed Address of your peer before the Relationship is established.                                        |
+| Relationship   | Sender          | Y         | `AUTO_ACCEPT`   | Company sends new customer his customer number.                                                                                                                                                                                                                       |
+| Relationship   | Recipient       | Y         | `USER_DECISION` | With this combination the **sender asks the recipient for the one-time permission** to write a Relationship Attribute once AND the **sender defined a value** which can either be accepted and stored, or rejected. Thus, the user cannot change the value by itself. |
+| Relationship   | `<empty>`       | Y         | `USER_DECISION` | An empty owner defaults to an Attribute with owner=`<recipient>`. This is needed for Requests inside of Relationship Templates, since you don’t know the Enmeshed Address of your peer before the Relationship is established.                                        |
+
+## ProposeAttributeRequestItem
+
+The ProposeAttributeRequestItem is a combination of [ReadAttributeRequestItem](#readattributerequestitem) and [CreateAttributeRequestItem](#createattributerequestitem). The sender would like to receive a correct Attribute from the peer, thinks it has a possible value but the peer might overrule this value with an existing or new one.
+
+To create a peer Attribute (owner = peer) with a value proposed by the sender, an Identity uses the ProposeAttributeRequestItem. A proposed value in this case means, that the recipient is allowed to change the value if accepting the request.
+
+### Examples
+
+-   Onboard an authenticated user and propose the known private Attributes
```

Comment:
Ich glaube, die Wörter "Onboard" und "authenticated" sind nur schwer verständlich für den durchschnittlichen Leser:
- wo onboarden? (wir meinen zwar Enmeshed, aber es könnte genauso gut alles andere sein)
- wo authenticated? (wir meinen zwar im Kundensystem, aber es könnte genauso gut alles andere sein)

Vielleicht wäre das hier besser: "Onboard an existing customer to Enmeshed and propose the known private Attributes, like its name or address."